### PR TITLE
Add new version of httpclient and openid4java

### PR DESCRIPTION
**Pull request by senthalan**
refer to mail "Updating common codec version to 1.10.wso2v1 in IS 5.6"

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: httpclient/4.3.1.wso2v3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        </dependency>
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
```
Comment (thusithathilina) [inline httpclient/4.3.1.wso2v3/pom.xml]: Snapshot distro should be removed

File: httpclient/4.3.6.wso2v2/pom.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        </dependency>
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
```
Comment (thusithathilina) [inline httpclient/4.3.6.wso2v2/pom.xml]: Snapshot should be removed

File: openid4java/1.0.0.wso2v4/pom.xml [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    </dependencies>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline openid4java/1.0.0.wso2v4/pom.xml]: snapshot should be removed

Comment (senthalan) [reply openid4java/1.0.0.wso2v4/pom.xml]: Removed

File: httpclient/4.3.6.wso2v2/pom.xml (hunk 2 above)
Comment (senthalan) [reply httpclient/4.3.6.wso2v2/pom.xml]: Removed

File: httpclient/4.3.1.wso2v3/pom.xml (hunk 1 above)
Comment (senthalan) [reply httpclient/4.3.1.wso2v3/pom.xml]: Removed

File: openid4java/1.0.0.wso2v4/pom.xml [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        <openid4java.version>1.0.0</openid4java.version>
+        <openid4java.export.version>1.0.0</openid4java.export.version>
+        <net.sourceforge.nekohtml.version>1.9.13</net.sourceforge.nekohtml.version>
+        <code.google.inject.version>[1.3.0,1.3.1)</code.google.inject.version>
+        <javax.crypto.version>0.0.0</javax.crypto.version>
+        <javax.net.version>0.0.0</javax.net.version>
+        <javax.servlet.version>[2.6.0,2.7.0)</javax.servlet.version>
+        <javax.xml.version>0.0.0</javax.xml.version>
+        <net.sf.ehcache.version>[1.5,1.6)</net.sf.ehcache.version>
+        <org.apache.commons.codec.version>[1.4,2.0)</org.apache.commons.codec.version>
```
Comment (pulasthi7) [inline openid4java/1.0.0.wso2v4/pom.xml]: add the suffix "range" for version ranges. do it for all ranges

**Pull request merged**